package.json

```diff
@@ -27,7 +26,6 @@
     "react-native-form": "^2.1.0",
     "react-native-google-places-autocomplete": "^1.1.10",
     "react-native-keyboard-aware-scroll-view": "^0.2.0",
-    "react-native-mapbox-gl": "^5.0.0",
```

Comment:
Looks like I forgot to return that back. I just take a lot of time for build, every time when I did rm -rf node_modules && npm i :) BTW, don't used by android I guess